README.adoc

```diff
@@ -72,11 +72,11 @@ See <<_configuring_clickhouse>> for details on ClickHouse's setup and configurat
 +
 . Additional software requirements:
 +
-On an illumos-based machine (Helios, OmniOS), if you want to run the real (non-simulated) Sled Agent to run actual VMs with Propolis, make sure your packages are up to date, and you have the `brand/sparse` package:
+On an illumos-based machine (Helios, OmniOS), if you want to run the real (non-simulated) Sled Agent to run actual VMs with Propolis, make sure your packages are up to date, and you have the `brand/omicron1/tools` package:
 +
 [source,text]
 ----
-pkg install brand/sparse
+pkg install brand/omicron1/tools
```

Comment:
Would this actually work on non-Helios? (https://pkg.omnios.org/r151030/core/en/search.shtml at least doesn't show it). If we're gonna start gating things to work on basically just Helios it might be worth throwing up a downloadable image somewhere since the helios source repo is not public.

Reply:
I added a link to the Helios ISOs in https://github.com/oxidecomputer/omicron/pull/797/commits/59b3cf5879413c1c12103c89354aa6a41578413e , which should be public.